packages-next/keystone/src/lib/createImagesContext.ts

```diff
@@ -5,6 +5,7 @@ import { v4 as uuid } from 'uuid';
 import fs from 'fs-extra';
 import { fromBuffer } from 'file-type';
 import imageSize from 'image-size';
```

Comment:
Do we still need this?

Reply:
@mitchellhamilton  should we revert things to using just sharp again, given that we still need to leverage sharp to get blurHash working?

Reply:
I think we need sharp for future ideas as well